daemon/src/device.rs

```diff
@@ -1472,11 +1611,13 @@ impl<'a> Device<'a> {
         Ok(changed)
     }
 
-    async fn load_effect_bank(&mut self, preset: EffectBankPresets) -> Result<()> {
-        // Send the TTS Message..
-        let preset_name = self.profile.get_effect_name(preset);
-        let tts_message = format!("Effects {}, {}", preset as u8 + 1, preset_name);
-        let _ = self.global_events.send(TTSMessage(tts_message)).await;
+    async fn load_effect_bank(&mut self, preset: EffectBankPresets, speak_tts: bool) -> Result<()> {
```

Comment:
As previously mentioned, speak_tts isn't needed anymore, It's currently only ever sent as true